examples/llama/tests/test_conversion.py

```diff
@@ -0,0 +1,192 @@
+# ruff: noqa: E402
+import json
+
+import pytest
+import torch
+from transformers import LlamaForCausalLM
+from utils import set_system_path
+
+set_system_path()
+
```

Comment:
would be better to add `__init__.py` file in `tests/` and `examples/` + rename from `from convert_weights` to `from examples.llama.convert_weights` in both converter script

Reply:
@3outeille, done! looks good.

Question: we took this code snippet from the doremi example. Should we try to clean this up from there as well? I don't want to break it though